doc/manuals/user/walkthrough_apiv2.md

```diff
@@ -709,17 +709,6 @@ Let's examine in detail the different elements included in the payload:
     not only if pressure changes, but if it changes within the range 700-800. This
     is an advanced topic, see the "Subscriptions" section in the
     [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/stable/).
--   The throttling element is used to specify a minimum
```

Comment:
This is not actually related with the initial notification deprecation, but I have taken the opportunity to remove this from the tutorial walk-through, as it is an advanced feature (and even problematic in some cases).

NTC (informative)